examples/stm32/xively_demo_stm32f4xx_nucleo_eth/SW4STM32/STM32F429ZI_NUCLEO_144/.cproject

```diff
@@ -5,7 +5,7 @@
 			<storageModule buildSystemId="org.eclipse.cdt.managedbuilder.core.configurationDataProvider" id="fr.ac6.managedbuild.config.gnu.cross.exe.debug.667843212" moduleId="org.eclipse.cdt.core.settings" name="Debug">
 				<macros>
 					<stringMacro name="XI_STMCUBE_PATH" type="VALUE_PATH_DIR" value="/Users/atigyi/Downloads/STM32Cube_FW_F4_V1.14.0"/>

```

Comment:
is there something else we can default this path to, or does it not really matter?